CHANGELOG.rst

```diff
@@ -1,6 +1,33 @@
 Changelog
 =========
 
+1.0.0 (2021-02-xx)
+------------------
+
+- [CHANGE] Arrow has **dropped support** for Python 2.7 and 3.5.
```

Comment:
Maybe add a CHANGE bullet saying that Arrow is now following semantic versioning (https://semver.org/)?